source/apps/export.txt

```diff
@@ -0,0 +1,210 @@
+.. _export-app:
+
+=============
+Export an App
+=============
+
+Overview
+--------
+
+You can download a directory of your App's configuration files by
+exporting them. This allows you to save your app's configuration in
```

Comment:
I 100% do not follow our rules for capitalizing `App` - why is it not capitalized here?
```suggestion
exporting them. This allows you to save your App's configuration in
```

Reply:
We used to be inconsistent about it and old habits die hard 😅 We should use the capitalized "App" in most cases - good catch!